universal-login-react/src/ui/commons/Assets.tsx

```diff
@@ -10,20 +10,40 @@ import './../styles/themes/Jarvis/assetsListThemeJarvis.sass';
 import {getIconForToken} from '../../core/utils/getIconForToken';
 import {useBalances} from '../hooks/useBalances';
 import {ThemedComponent} from './ThemedComponent';
+import DisplayTokens from './DisplayTokens';
+import Spinner from './Spinner';
+import {classForComponent} from '../utils/classFor';
+import {IToken, getErc721Tokens} from '../../core/utils/getErc721Tokens';
+
+export type assetState = 'COINS' | 'TOKENS' | 'LOADING';
 
 export interface AssetsProps {
   deployedWallet: DeployedWallet;
 }
 
 export const Assets = ({deployedWallet}: AssetsProps) => {
   const [tokenDetailsWithBalance] = useBalances(deployedWallet);
+  const [currentState, setCurrentState] = useState('COINS' as assetState);
```

Comment:
You can try useState<assetState>('COINS');

Reply:
`'COINS' as AssetState` is better option in my opinion